department-of-veterans-affairs/va.gov-cms
# VACMS-4035: Add removal logic for service locations.

## Description
See #4035 

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/105559046-24831e80-5cde-11eb-9dfc-75fdf1331323.png)

## QA steps
- As an administrator, go to `/node/1045/edit`
- Visually verify that there are now remove buttons for each paragraph

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
